ground/gcs/src/plugins/config/inputchannelform.h

```diff
@@ -12,7 +12,7 @@ class inputChannelForm : public ConfigTaskWidget
     Q_OBJECT
 
 public:
-    explicit inputChannelForm(QWidget *parent = 0,bool showlegend=false);
+    explicit inputChannelForm(QWidget *parent = 0,bool showlegend=false, bool showSlider=true);
```

Comment:
Spacing on args (I know there's tons of violations here, but if you're touching the line it should probably be fixed or at least consistent).